<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="8" total="12" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-17T12:55:24 IST" name="Suite" finished-at="2025-08-17T12:55:43 IST" duration-ms="18603">
    <groups>
    </groups>
    <test started-at="2025-08-17T12:55:24 IST" name="chromeTest" finished-at="2025-08-17T12:55:37 IST" duration-ms="12492">
      <class name="com.automation.test.LoginPageTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:24 IST" name="setup" finished-at="2025-08-17T12:55:30 IST" duration-ms="5535" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testInvalidPassword()[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:30 IST" name="testInvalidPassword" finished-at="2025-08-17T12:55:31 IST" duration-ms="946" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message not displayed for invalid password. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message not displayed for invalid password. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.automation.test.LoginPageTest.testInvalidPassword(LoginPageTest.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidPassword -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:31 IST" name="quit" finished-at="2025-08-17T12:55:31 IST" duration-ms="185" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:31 IST" name="setup" finished-at="2025-08-17T12:55:36 IST" duration-ms="4950" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testValidLogin()[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:36 IST" name="testValidLogin" finished-at="2025-08-17T12:55:37 IST" duration-ms="231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidLogin -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.automation.test.LoginPageTest@471a9022]" started-at="2025-08-17T12:55:37 IST" name="quit" finished-at="2025-08-17T12:55:37 IST" duration-ms="193" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.automation.test.LoginPageTest -->
    </test> <!-- chromeTest -->
    <test started-at="2025-08-17T12:55:24 IST" name="firefoxTest" finished-at="2025-08-17T12:55:43 IST" duration-ms="18603">
      <class name="com.automation.test.LoginPageTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:24 IST" name="setup" finished-at="2025-08-17T12:55:31 IST" duration-ms="6601" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testInvalidPassword()[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:31 IST" name="testInvalidPassword" finished-at="2025-08-17T12:55:32 IST" duration-ms="979" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message not displayed for invalid password. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message not displayed for invalid password. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.automation.test.LoginPageTest.testInvalidPassword(LoginPageTest.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidPassword -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:32 IST" name="quit" finished-at="2025-08-17T12:55:34 IST" duration-ms="2152" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:34 IST" name="setup" finished-at="2025-08-17T12:55:41 IST" duration-ms="7036" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testValidLogin()[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:41 IST" name="testValidLogin" finished-at="2025-08-17T12:55:42 IST" duration-ms="821" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidLogin -->
        <test-method is-config="true" signature="quit()[pri:0, instance:com.automation.test.LoginPageTest@e3c0e40]" started-at="2025-08-17T12:55:42 IST" name="quit" finished-at="2025-08-17T12:55:43 IST" duration-ms="813" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.automation.test.LoginPageTest -->
    </test> <!-- firefoxTest -->
  </suite> <!-- Suite -->
</testng-results>
